programs/uxd/tests/integration_tests/api/program_credix/binaries/dump.sh

```diff
@@ -1,5 +1,5 @@
 solana config set --url https://api.devnet.solana.com
-solana program dump crdszSnZQu7j36KfsMJ4VEmMUTJgrNYXwoPVHUANpAu executable-devnet.so
+solana program dump crdRi38zEhQdzpsxnKur73WHBM9BSvXMSfGcbLyJCdP executable-devnet.so
```

Comment:
call it executable-pre-mainnet?

Reply:
Well, it does run on the devnet tho T_T

Reply:
yea let's keep the name then coz it's the only legit devnet program we should use going forward